src/components/LoanFinding/QuickFiltersSection.vue

```diff
@@ -110,7 +114,7 @@ export default {
 			loanSearchState: {
 				pageOffset: 0,
 				pageLimit: 6,
-				sortBy: 'amountLeft'
+				sortBy: this.enableAlmostFundedRow ? 'expiringSoon' : 'amountLeft'
```

Comment:
I forget, can the `computed` be used here as well?

Reply:
I think it only gets set on initialization but would not be reactive for subsequent changes...

Reply:
I think that would be the same as this line, but I forget if `computed` is accessible yet in `data`.

Reply:
Unfortunately not, this is the main reason:
`The computed property cannot be used in `data()` because it is before initialization.`

Reply:
Okay then, I was wondering, thanks for checking!